### DISCUSSION TITLE
what are max speeds?

### DISCUSSION THREAD

**Discussion by b1ek** [meshtastic/Discussions#113] [Firmware]
i'm considering building a small meshtatic network around my block to be able to stream video from my jellyfin server at home to a cafe where i usually hang out at. it's about 10-15 meters away with no obstacles other than a few drywalls and glass windows. is it theoretically possible to get at least 20 mpbs out of it?

is it even worth it in the long run? i could get an unlimited gigabit connection on my mobile for about 50 eur a month, which is honestly not that much for me

**Comment by ianmcorvidae** [comment]
Video streaming will absolutely not work over Meshtastic. The strength of LoRa is range, which is at the expense of speed. There's a table of the theoretical maximum data rates at https://meshtastic.org/docs/overview/radio-settings/#presets -- the fastest is only about 20 kilobits per second, and that's without any hops. If you can get the range, WiFi meshing could work for you, but at a guess, better to buy the phone Internet.
